src/Command/QueueCleanCommand.php

```diff
@@ -25,12 +25,12 @@ protected function configure()
     {
         $this
             ->setName('queue:clean')
-            ->setDescription('Cleans the SQS queue through purging. Asynchronous.');
+            ->setDescription('Cleans the SQS queue through purging.');
```

Comment:
A way to express the fact that the queue is not immediately purges, but it takes up to 60 seconds after the command returns for this to happen?